tools/msvc/sbclang.targets

```diff
@@ -828,6 +814,7 @@
     <Link Condition="'$(UseWinObjCCRT)' == 'true'">
       <!-- Ensure we pick up operator new from our project rather than the C runtime -->
       <ForceSymbolReferences>??2@YAPAXI@Z</ForceSymbolReferences>
+      <AdditionalOptions>/DISALLOWLIB:libucrt.lib /DISALLOWLIB:libucrtd.lib /nodefaultlib:ucrt /nodefaultlib:ucrtd /nodefaultlib:vccorlib /nodefaultlib:msvcrt /nodefaultlib:vccorlibd /nodefaultlib:msvcrtd /defaultlib:winobjcrt winobjcrt.lib %(AdditionalOptions)</AdditionalOptions>
```

Comment:
putting a .lib in the additionaloptions is scary; it can't be an Input? I worry that a change in link option order would break this.